lib/servers/iXonServer/qt4reactor.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2001-2008 Twisted Matrix Laboratories.
```

Comment:
to clarify on this dylan and i think that using the same file could shove things into the same qt event loop and possibly slow things down...unknown though..